src/main/java/com/checkout/marketplace/MarketplaceClient.java

```diff
@@ -26,6 +26,8 @@ public interface MarketplaceClient {
 
     CompletableFuture<CreateTransferResponse> initiateTransferOfFunds(CreateTransferRequest createTransferRequest);
 
+    CompletableFuture<CreateTransferResponse> initiateTransferOfFunds(CreateTransferRequest createTransferRequest, String idempotencyKey);
```

Comment:
I was about to say, isn't more easier to add `String idempotencyKey = null` 
just kidding.